webapp/server.py

```diff
@@ -0,0 +1,39 @@
+# Adapted from Software Design's serve_my_exe.py program for using
+# the SD_app React app.
+
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from subprocess import *
+import sys
+import logging
+
+exe_name = ""
```

Comment:
I don't have a good way to avoid this now, but could I ask you to file an issue about removing it after we merge this PR?

Reply:
Sure